manager.py

```diff
@@ -17,12 +17,19 @@ class TwitchManager:
     SECONDS_BETWEEN_UPDATE_STREAMS = 60
     SECONDS_BETWEEN_CREATE_BOTS = 15
 
-    def __init__(self, channels_amount, channels, log_filename=None):
+    def __init__(self, channels_amount, channels, log_filename=None, users_whitelist=None):
         self.bots = []
         self.channels_amount = channels_amount
         self.log_filename = log_filename
         self.channels = channels
-        self.db_logger = DatabaseLogger(settings.DATABASE['HOST'],
+        if settings.USERS is not None and len(settings.USERS) is not 0:
```

Comment:
You do not need this if condition here. If `settings.USERS` is `None`, it will do exactly what it should.